helm-chart/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+{{- if and .Values.frontend.enabled .Values.frontend.externalService }}
+Get the application URL by running these commands:
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ .Values.frontend.name }}-external'
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ .Values.frontend.name }}-external --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
+  echo http://$SERVICE_IP
+{{- end }}
```

Comment:
Praise:
This user experience is cool. I love that we can apply it conditionally (e.g., based on `.Values.frontend.externalService`).